### ISSUE TITLE
Can we setup Security Onion in Amazon AWS to monitor AWS instances?

### ISSUE THREAD

**Issue by jeewanthasas** [Snorby/snorby#444]
Can we setup Security Onion in Amazon AWS to monitor AWS instances?

**Comment by shadowbq** [comment]
AWS does not a have a layer 2. You can not would need to route traffic through snort.. which is not what snort does.

**Comment by shadowbq** [comment]
You could use suricate inline.. https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Setting_up_IPSinline_for_Linux

**Comment by mihail-velikov** [comment]
Yes, it is possible to setup Security Onion in AWS although it is in experimental state:
https://github.com/Security-Onion-Solutions/security-onion/wiki/CloudClient
